src/Sulu/Bundle/PageBundle/Tests/Unit/Admin/PageAdminTest.php

```diff
@@ -124,9 +124,29 @@ public function testGetConfigWithVersioning()
             true
         );
 
+        $webspace1 = $this->prophesize(Webspace::class);
+        $webspace1->getName()->willReturn('second webspace');
+
+        $webspace2 = $this->prophesize(Webspace::class);
+        $webspace2->getName()->willReturn('first webspace');
+
+        $this->webspaceManager->getWebspaceCollection()->willReturn(new WebspaceCollection([
+            $webspace1->reveal(),
+            $webspace2->reveal(),
+        ]));
+
+        $this->teaserProviderPool->getConfiguration()->willReturn([]);
+
         $config = $admin->getConfig();
 
-        $this->assertEquals(true, $config['versioning']);
+        $this->assertEquals(
+            [
+                'teaser' => [],
+                'versioning' => true,
+                'webspaces' => [$webspace1->reveal(), $webspace2->reveal()],
```

Comment:
Shouldn't `$webspace2` be listed here first, because `f` comes before `s` in the alphabet? :thinking:

Reply:
you are right 🙈 the problem was that we keep the array key because it is set to the webspace-key. adjusted the test and added an additional check